book/modules/module11-exercises.tex

```diff
@@ -230,8 +230,8 @@
 
 				\item False. This is false whenever $m>n$ (follows from rank-nullity
 					theorem) Take for example: $T:\R^{3}\to\R^{2}$ induced
-					by the matrix $A=\mat{1&0&0\\0&1&0}$;\\ $\Rank(T)=\Dim(\Col
-					A)=2$ $\Nullity(T) = \Nullity(A)=1$
+					by the matrix $A=\mat{1&0&0\\0&1&0}$;\\ We have that $\Rank(T)=\Dim(\Col
```

Comment:
Lower case `W` following a semicolon.

Reply:
Submitted a new version with the lower case